packages/cli/src/main.ts

```diff
@@ -1,8 +1,100 @@
 #!/usr/bin/env node
-import { Command } from 'commander';
+import { Command, createOption, Option } from 'commander';
 import { node } from './commands/node';
+import { TaskParam, TaskArguments, initialKuai, paramTypes, KuaiArguments } from '@kuai/core';
 
-const program = new Command();
+const KUAI_GLOBAL_PARAMS: Array<TaskParam> = [
+  {
+    name: 'config',
+    description: 'A Kuai config file.',
+    type: paramTypes.path,
+    isFlag: false,
+    isOptional: true,
+  },
+];
 
-program.addCommand(node);
-program.parse();
+function parseTaskParams(param: TaskParam<TaskArguments>): Option {
+  const type = (() => {
+    if (param.type.name === 'boolean') {
+      return '';
+    }
+
+    return ` <${param.type.name}>`;
+  })();
+
+  return createOption(`--${param.name}${type}`, param.description);
+}
+
+const main = async () => {
+  const program = new Command();
+
+  KUAI_GLOBAL_PARAMS.map((param) => parseTaskParams(param)).forEach((option) => program.addOption(option));
+
+  program.addCommand(node);
+
+  const args: KuaiArguments = (() => {
+    const result: KuaiArguments = {};
+
+    const configPathFlagIndex = process.argv.slice(0, -1).lastIndexOf('--config');
+
+    if (configPathFlagIndex !== -1) {
+      Object.assign(result, { configPath: process.argv[configPathFlagIndex + 1] });
+    }
+
+    return result;
+  })();
+
+  const ctx = await initialKuai(args);
+  const env = ctx.getRuntimeEnvironment();
+  Object.values(env.tasks)
+    .filter((task) => !task.isSubtask)
+    .map((task) => {
```

Comment:
Use `forEach` instead of `map` because `map` implies the returned value is useful.

A warning will be marked by some code analysis tools
![image](https://user-images.githubusercontent.com/7271329/203918426-295424b6-6df0-46d5-ae9b-08da2da673b6.png)

Same issue at, https://github.com/ckb-js/kuai/pull/32/files#diff-79567c60a701dd642531b599dee81fc4b0829920af74fd352277f88f65693527R60 https://github.com/ckb-js/kuai/pull/32/files#diff-79567c60a701dd642531b599dee81fc4b0829920af74fd352277f88f65693527R71, https://github.com/ckb-js/kuai/pull/32/files#diff-79567c60a701dd642531b599dee81fc4b0829920af74fd352277f88f65693527R88